Lib/test/test_regrtest.py

```diff
@@ -1747,7 +1747,7 @@ def test_other_bug(self):
     @support.cpython_only
     def test_uncollectable(self):
         try:
-            import _testcapi
+            import _testcapi  # noqa: F401
         except ImportError:
             raise unittest.SkipTest("requires _testcapi")
```

Comment:
`import_module()` can be used here.

Reply:
Ok, I created https://github.com/python/cpython/pull/120680 for test_regrtest.